src/main/java/com/faforever/client/vault/replay/LiveReplayController.java

```diff
@@ -68,18 +68,17 @@ public LiveReplayController(GameService gameService, UiService uiService, I18n i
   }
 
   public void initialize() {
-    Comparator gameSorter = Comparator.comparing(
-        (Game game) -> game.getStartTime(),
-        Comparator.nullsFirst(Comparator.naturalOrder()))
-        .reversed();
-    initializeGameTable(gameService.getGames()
-                    .filtered(game -> game.getStatus() == GameStatus.PLAYING)
-                    .sorted(gameSorter));
+    initializeGameTable(gameService.getGames().filtered(game -> game.getStatus() == GameStatus.PLAYING));
```

Comment:
We could move sorting and filtering together (here or in `initializeGameTable`).

Reply:
We Could